public_html/templates/index.html

```diff
@@ -1,100 +1,118 @@
 {% extends "base.html" %}
+{% import _self as macros %}
+
+{% macro reviewer_node( row ) %}
+    <div class="status-div--reviewer">
```

Comment:
[BEM](http://getbem.com/) methodology, which I tend to do instinctively. Going by the [MediaWiki coding conventions](https://www.mediawiki.org/wiki/Manual:Coding_conventions/CSS) it looks like we don't do this, so I can unlearn it :)